data/jibauni/jibauni jobs.txt

```diff
@@ -0,0 +1,391 @@
+# Copyright (c) 2024 by Loymdayddaud
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Jibauni Pilgrims [0]"
```

Comment:
If the Jibauni have an annual pilgrimage analogous to something like the Hajj, there could be another Jibauni pilgrim job with a higher payout for a portion of the year implemented like the `Earth Day` near earth jobs.

```to offer
		or
			and
				random < 70
				month == 4
				day > 22
			and
				random < 50
				month == 5
			and
				random < 30
				month == 6
				day <= 22
```

Reply:
I had wondered about that, but I'm not sure it would really fit in with what the Jibauni are supposed to believe.
For one thing, the Jibauni discovered Hajay/Lamalmuq fairly recently - and their religion has had a "holy place" since long before that.
And also, it's perfectly fine for Jibauni to _not_ go, unlike the Haj, which is required before death for Muslims.
I just thought it would be nice to have an alien species in Endless Sky that was more religious.